### PR TITLE
bradl3yC - 9382 - debt letters entry

### PR DESCRIPTION
**Pull request by bradl3yC**
## Description
- [X] -  Create entry point for new Debt Letters work

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/debt-letters/reducers/index.js [hunk 1]
```diff
@@ -0,0 +1 @@
+export default {};
```
Comment (LinnJS) [inline src/applications/debt-letters/reducers/index.js:1]: Is this intentional as boilerplate?

**Review by LinnJS — APPROVED** [review]

File: src/applications/debt-letters/reducers/index.js (hunk 1 above)
Comment (bradl3yC) [reply src/applications/debt-letters/reducers/index.js:1]: @LinnJS  Intentional - I am adding the reducer / actions in the next PR

**Review by maharielrosario — APPROVED** [review]

**Review by LinnJS — APPROVED** [review]

**Comment by bradl3yC** [comment]
@department-of-veterans-affairs/frontend-review-group small PR to add entry point for upcoming debt-letters work

File: src/applications/debt-letters/app-entry.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import 'platform/polyfills';
+import './sass/debt-letters.scss';
+import DebtLettersWrapper from './components/DebtLettersWrapper';
+
+import startApp from '../../platform/startup';
```
Comment (jhonnyoddball) [inline src/applications/debt-letters/app-entry.jsx:6]: Please fix this and use the alias instead
```suggestion
import startApp from 'platform/startup';
```

**Review by jhonny218 — APPROVED** [review]
LGTM

**Pull request merged**